backends-velox/src/main/scala/io/glutenproject/backendsapi/velox/TestApiImpl.scala

```diff
@@ -0,0 +1,415 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.glutenproject.backendsapi.velox
+
+import io.glutenproject.backendsapi.TestApi
+import io.glutenproject.backendsapi.TestApi.{SparkVersion, SparkVersion32, SparkVersion33}
+
+class TestApiImpl extends TestApi {
+
+  /**
+   * List of supported cases to run with Velox backend, in lower case. Please add to the supported
+   * list after enabling a sql test.
+   */
+  override def getSupportedSQLQueryTests(sparkVersion: SparkVersion): Set[String] = {
+    sparkVersion match {
+      case SparkVersion32() =>
+        SUPPORTED_SQL_QUERY_LIST_SPARK32
+      case SparkVersion33() =>
+        SUPPORTED_SQL_QUERY_LIST_SPARK33
+    }
+  }
+
+  override def getOverwriteSQLQueryTests(sparkVersion: SparkVersion): Set[String] = {
+    sparkVersion match {
+      case SparkVersion32() =>
+        OVERWRITE_SQL_QUERY_LIST_SPARK32
+      case SparkVersion33() =>
+        OVERWRITE_SQL_QUERY_LIST_SPARK33
+    }
+  }
+
+  private val SUPPORTED_SQL_QUERY_LIST_SPARK32: Set[String] =
```

Comment:
@rui-mo Do you mean some of the filenames listed here might not specifying the subdirectory? If so, we should add the missing subdirectories.